src/main/java/teammates/ui/controller/InstructorHomePageAction.java

```diff
@@ -59,15 +57,21 @@ public ActionResult execute() throws EntityDoesNotExistException {
         data.instructors = new HashMap<String, InstructorAttributes>();
         data.numberOfPendingComments = new HashMap<String, Integer>();
         
-        for(CourseSummaryBundle course: data.courses) {
+        for (CourseSummaryBundle course : data.courses) {
             String courseId = course.course.id;
-            InstructorAttributes instructor = logic.getInstructorForGoogleId(courseId, account.googleId);
+            InstructorAttributes instructor = logic.getInstructorForGoogleId(
+                                            courseId, account.googleId);
             data.instructors.put(courseId, instructor);
-            int numberOfPendingCommentsForThisCourse = logic.getCommentsForSendingState(courseId, CommentSendingState.PENDING).size() 
-                    + logic.getFeedbackResponseCommentsForSendingState(courseId, CommentSendingState.PENDING).size();
+            
+            int numberOfPendingCommentsForThisCourse = logic.getCommentsForSendingState(
+                                            courseId, CommentSendingState.PENDING).size() 
+                                            + logic.getFeedbackResponseCommentsForSendingState(
+                                            courseId, CommentSendingState.PENDING).size();
```

Comment:
Hard to see what's going on here. Extract the second parameter into a local variable? i.e. 'introduce explaining variable' refactoring.